css/style.css

```diff
@@ -15,5 +15,113 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+ h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 2rem 5rem;
+}
+
+.header,
+.content,
+.footer {
+  padding: 0 1.5rem;
+  max-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2rem 5rem;
+}
```

Comment:
nit: rather than applying these stylings to all 3 classes, you could instead apply them to a single class (e.g. `.text-block`), and then apply that class to each of your header/content/footer (e.g. `<div class="header text-block">`) - that way you're not coupling the styling of the header/footer/content to each other